### PR TITLE
Add Media Session API

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Source: https://groups.google.com/a/chromium.org/forum/#!topic/blink-dev/xIpJ_303phw

FYI @mounirlamouri @xxyzzzq

### CODE REVIEW

File: status.json [hunk 1]
```diff
@@ -6000,5 +6000,12 @@
     "wpd": "",
     "id": 6170807885627392,
     "uservoiceid": null
+  },
+  {
+    "name": "Media Session",
+    "category": "Multimedia",
+    "link": "https://wicg.github.io/mediasession",
+    "summary": "Enable websites to customize media metadata so they can control the notification and lock screen UI. Also, allow them to control media in the page from outside the page, such as platform UIs and media keys.",
+    "standardStatus": "Editor's draft"
```
Comment (alrra) [inline status.json]: @beaufortfrancois Can you add [`"id": 5639924124483584`](https://www.chromestatus.com/feature/5639924124483584). Thanks!

**Comment by beaufortfrancois** [comment]
@alrra Done ;)

**Comment by alrra** [comment]
Will check internally to see what is Edge's status, and then merge this.

**Comment by alrra** [comment]
Thanks, @beaufortfrancois!

**Comment by beaufortfrancois** [comment]
In https://developer.microsoft.com/en-us/microsoft-edge/platform/status/mediasession/?q=media%20session, it says "In Development in Opera" and "In Development in Safari" which is not true.
Do you mind checking out what is wrong?

Thanks @alrra!

**Comment by alrra** [comment]
> In https://developer.microsoft.com/en-us/microsoft-edge/platform/status/mediasession/?q=media%20session, it says "In Development in Opera" and "In Development in Safari" which is not true.

@beaufortfrancois The data for browsers other than Microsoft Edge (and Internet Explorer) comes from the [Chromium Dashboard](https://www.chromestatus.com/features). 



Mind opening an issue [there](https://github.com/GoogleChrome/chromium-dashboard/issues/new)? Thanks!

**Pull request closed without merging**